main.py

```diff
@@ -1,7 +1,7 @@
 """
 Upload transcribed audio files to DocumentCloud using Whisper
 """
-
+from __future__ import unicode_literals
```

Comment:
Were you trying to run this on Python 2?  This line should be removed, it is unnecessary for running under Python 3

Reply:
Yea, probably leftover code